src/alert-prelude.c

```diff
@@ -478,6 +575,157 @@ static int PacketToDataV6(const Packet *p, const PacketAlert *pa, idmef_alert_t
     SCReturnInt(0);
 }
 
+/**
+ * \brief Convert JSON object to Prelude additional data with
+ * the right type of data.
+ *
+ * \return 0 if ok
+ */
+int JsonToAdditionnalData(const char * key, json_t * value, idmef_alert_t *alert ) {
```

Comment:
this use of json_t without HAVE_JANSSON ifdefs looks suspicious to me. Will this code compile if libjansson is not available?